nexmark-flink/src/main/resources/conf/sql-client-defaults.yaml

```diff
@@ -53,21 +53,6 @@ execution:
     type: fallback
 
 #==============================================================================
-# Configuration options
-#==============================================================================
-
-# Configuration options for adjusting and tuning table programs.
-
-# A full list of options and their default values can be found
-# on the dedicated "Configuration" web page.
-
-# A configuration can look like:
-configuration:
-  table.exec.mini-batch.enabled: true
-  table.exec.mini-batch.allow-latency: 2s
-  table.exec.mini-batch.size: 50000
-  table.optimizer.distinct-agg.split.enabled: true
-#==============================================================================
```

Comment:
Could you keep this configuration? Because some old versions only support reading SQL configuration from `sql-client.yaml`.

Reply:
It makes sense to me. I have reset this configuration.